.ci/docker/common/install_base.sh

```diff
@@ -15,6 +15,9 @@ install_ubuntu() {
   elif [[ "$UBUNTU_VERSION" == "22.04"* ]]; then
     cmake3="cmake=3.22*"
     maybe_libiomp_dev=""
+  elif [[ "$UBUNTU_VERSION" == "24.04"* ]]; then
+    cmake3="cmake=3.28*"
```

Comment:
For my edification: does Ubuntu24.04 *need* a higher CMake version, or is this the lowest version of CMake available for Ubuntu 24.04?

Reply:
This version is basically to satisfy only the `apt install ` command.
The cmake used is referred from `/opt/conda/envs/py_3.12/bin/cmake` with 3.26 installed from conda which is same in UB 22.04 also.

Reply:
Yeah, what Pruthvi said. I haven't been able to make it work with anything lower than this.

Reply:
Okay, so it seems the answer is "this is the lowest version of CMake available for Ubuntu 24.04".